plugins/katello/nightly/user_guide/content_view_import_export/index.md

```diff
@@ -100,38 +100,42 @@ The `hammer content-view version export` command gathers information about a con
+</div>
 
 The hammer command will also create a tar file that contains all of the repositories listed. The final result of the command is a tar file that contains two files: the json, and an inner tar file with all of the repositories. This tar file can be copied to a USB key and used for the `import` command.
 
-:::info
-_NOTE_: Older versions of Katello relied on the Pulp `export_distributor` and `group_export_distributor` to create an ISO image with the yum repositories. Katello would start a server-side task, create an ISO, and then copy the ISO to `/var/lib/pulp/katello-export`. This process could take many hours and hundreds of GB of disk space for temporary files. Users can now use hammer to create the tar file. This lets us avoid creating temporary copies of extremely large files.
-:::
+<div class="alert alert-info" markdown="1">
+**Note**
 
-### Hammer import
+Older versions of Katello relied on the Pulp **_export_distributor_** and **_group_export_distributor_** to create an ISO image with the yum repositories. Katello would start a server-side task, create an ISO, and then copy the ISO to **_/var/lib/pulp/katello-export_**. This process could take many hours and hundreds of GB of disk space for temporary files. Users can now use hammer to create the tar file. This lets us avoid creating temporary copies of extremely large files.
+</div>
 
-The `hammer content-view version import` command uses the tar file created from the `export` command to create a content view version with the same data. It will create a content view version with the same major and minor version numbers, and the same repositories with the same packages and errata.
+### Hammer import
 
-Before you run the `import` command for the first time, you will need to create the same products on the importing Katello that you had on the exporting Katello. This is a step you will only need to do once. You will also need to create the same content view, with same label. Again, you will only need to do this once.
+* The `hammer content-view version import` command uses the tar file created from the `export` command to create a content view version with the same data. It will create a content view version with the same major and minor version numbers, and the same repositories with the same packages and errata.
 
-The `import` command will synchronize the packages from the export tar file into Library. It will then call the `/publish` API and create a new content view version using those packages.
+* Before you run the `import` command for the first time, you will need to create the same products on the importing Katello that you had on the exporting Katello. This is a step you will only need to do once. You will also need to create the same content view, with same label. Again, you will only need to do this once.
```

Comment:
This section as bullets doesn't follow for me. I think they worked better as paragraphs.